circle.yml

```diff
@@ -1,7 +1,17 @@
 machine:
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
   node:
     version: 6.9
 
+dependencies:
+  override:
+    - yarn
```

Comment:
@hbetts Good idea

CircleCI runs v0.18.1 by default, so needed to manually upgrade it.

But then I noticed this problem, only fixed in `yarn` a day ago: https://github.com/yarnpkg/yarn/pull/2663

I'm considering whether to merge this anyway, and hope it gets included into v0.20.0 once it's released.